deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/pom.xml

```diff
@@ -48,6 +48,12 @@
         <version>${threadly.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
Also try to force a dependency on the same version of JavaCPP as used in the rest of Deeplearning4j.